hw/ip/edn/doc/edn_top_diag.svg

```diff
@@ -3,7 +3,7 @@
 <!-- Generated by Microsoft Visio, SVG Export edn_top_diag.svg Page-1 -->

```

Comment:
Wait, I'm confused.  This looks like you are putting the broadcast back into this figure.  What is your intent here?

Reply:
Yes, this is in error. Need to update this diagram first.

Reply:
During the hand-over process of this PR from @martin-lueker to me, the source (visio) was intack, but the svg file was not re-generated. This has been corrected.